tests/test_noise.py

```diff
@@ -27,13 +27,13 @@
 ALL_NOISES = list(DigitalNoiseType)
 
 
-def noise_instance(noise_type: DigitalNoiseType) -> DigitalNoiseInstance:
+def noise_instance(noise_type: DigitalNoiseType, zero: bool = False) -> DigitalNoiseInstance:
```

Comment:
Yeah, not sure `zero` is the best name but I cant think of anything better. Maybe, `zero_proba` ?